tools/docker/cpu_tests.Dockerfile

```diff
@@ -20,7 +20,8 @@ RUN python configure.py
 RUN pip install -e ./
 RUN --mount=type=cache,id=cache_bazel,target=/root/.cache/bazel \
     bash tools/install_so_files.sh
-RUN pytest -v -n auto --durations=25 --cov=tensorflow_addons ./tensorflow_addons/
+RUN pytest -v -n auto --durations=25 --doctest-modules ./tensorflow_addons \
+    --cov=tensorflow_addons ./tensorflow_addons/
```

Comment:
I think it makes sense to also run doctest on cpu_tests as docstring is also a part of codes.

Reply:
Agree